test/test_nn.py

```diff
@@ -1507,6 +1507,16 @@ def test_Sequential_delitem(self):
         del n[1::2]
         self.assertEqual(n, nn.Sequential(l1, l3))
 
+    def test_Sequential_append(self):
+        l1 = nn.Linear(10, 20)
+        l2 = nn.Linear(20, 30)
+        l3 = nn.Linear(30, 40)
+        l4 = nn.Linear(40, 50)
+        n = nn.Sequential(l1, l2, l3)
+        n2 = n.append(l4)
+        self.assertEqual(n, nn.Sequential(l1, l2, l3, l4))
+        self.assertEqual(n2, nn.Sequential(l1, l2, l3, l4))
```

Comment:
nit: maybe also test chaining via the returned self? e.g. `n.append(...).append(...)` Not super important but would be nice

Reply:
Agreed, I added a line below that checks for chaining!